src/face_position_cache.cpp

```diff
@@ -80,7 +81,8 @@ const std::vector<v3s16> &FacePositionCache::generateFacePosition(u16 d)
 	// Take blocks in all sides, starting from y=0 and going +-y
 	for (s16 y = 0; y <= d - 1; y++) {
 		// Left and right side, including borders
-		for (s16 z =- d; z <= d; z++) {
+		c.reserve(((d * 2) + 1) * 2); // formula ((d - (-d)) + 1) * 2
```

Comment:
not sure if it makes sense to optimize this, since it only ever runs exactly once

Reply:
> not sure if it makes sense to optimize this, since it only ever runs exactly once

@sfan5, if you strive for lowest load time and have killer feature like low requirements (aka PC 90-00s), then it makes sense. This does not spoil code style much.

Reply:
> if you strive for lowest load time and have killer feature like low requirements (aka PC 90-00s), then it makes sense.

I'm yet to see evidence of this for the face position cache specifically. This is most probably *very, very* negligible.

And applying such premature optimizations consistently would spoil the code style and waste development time significantly.